tests/test_playbook_runs.py

```diff
@@ -27,6 +27,19 @@ def get_molecule_scenario():
     return get_server_host()
 
 
+def get_disabled_test(group_name, test_name):
+    disabled_modules = os.environ.get("DISABLED_MODULES", "").split(":")
+    disabled_tests = os.environ.get("DISABLED_TESTS", "").split(":")
+
+    test = (
+        group_name in disabled_modules
+        or ("%s/%s" % (group_name, test_name)) in disabled_tests
+    )
+    if test:
+        print("%s/%s" % (group_name, test_name))
```

Comment:
I don't think we need a print here. If you run `pytest -rs` you will see the reason.

Reply:
debug left behind. I'll remove.